outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "webapp_url" {
+  value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "deploy_webhook_url" {
```

Comment:
Note that this output value contains a password - Terraform lets us [mark it with the sensitive flag](https://www.terraform.io/docs/language/values/outputs.html#sensitive-suppressing-values-in-cli-output) to ensure that it's treated as such and not logged to the console

Reply:
done